### PR TITLE
Bump k-charted, Sparkline component moved to k-charted

### PR DESCRIPTION
**Pull request by jotak**
Also fixes https://github.com/kiali/kiali/issues/2153, https://github.com/kiali/kiali/issues/2125 and https://github.com/kiali/kiali/issues/2088

This k-charted upgrade impacts both teams:

- @hhovsepy / https://github.com/kiali/kiali/issues/2125
This is to allow multiple colored squares being show in a single tooltip, when hovering series on crossing datapoint. E.g. :

*Before:*
Capture d’écran de 2020-02-10 10-45-41

*After:*
Capture d’écran de 2020-02-10 10-45-55

- @mattmahoneyrh on saturn's side, this is mainly some maintenance tasks, the code for Sparkline charts has been moved & the temporary hack https://github.com/kiali/kiali-ui/pull/1618 replaced with a more durable solution ; I don't expect any visual change but we should make sure there's no regression on all graph side panel charts.

### CODE REVIEW

**Comment by jotak** [comment]
DNM: waiting for saturn input

**Comment by jshaughn** [comment]
> DNM: waiting for saturn input

cc @israel-hdez

File: src/utils/Graphing.ts [hunk 1]
```diff
@@ -1,46 +1,9 @@
...
-    if (color) {
-      legendItem.symbol = { fill: color };
-    }
-    return {
-      datapoints: datapoints,
-      legendItem: legendItem,
-      color: color
-    };
+  toVCLine(dps: Datapoint[], name: string, color?: string): VCLine {
+    return toVCLine(toVCDatapoints(dps, name), { name: name, color: color });
```
[the remark this answers is not in the document]
Comment (jotak) [reply src/utils/Graphing.ts]: oh, good catch! not intentional at all. thanks!

**Comment by jotak** [comment]
thanks reviewers!

**Pull request merged**